src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,67 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  onSelectedOption: (option: string) => void;
+  onInputChange: (query: string) => void;
+  inputValue: string,
+  onClearQuery: ()=> void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+const SELECTED_OPTIONS = ['All', 'Active', 'Completed'];
```

Comment:
you can create `enum FilterOptions` and use it everywhere. it will help you to avoid errors and typos